recipe/meta.yaml

```diff
@@ -25,18 +27,22 @@ requirements:
 test:
   imports:
     - chardet
+  requires:
+    - pip
   commands:
+    - pip check
     - chardetect --help
 
 about:
   home: https://github.com/chardet/chardet
-  license: LGPL2
+  license: LGPL-2.1-only 
   license_family: GPL
   license_file: LICENSE
+  license_url: https://github.com/chardet/chardet/blob/main/LICENSE
   summary: Universal character encoding detector
   description: |
     chardet is a character encoding auto-detector in Python.
-  doc_url: http://chardet.readthedocs.org/
+  doc_url: https://chardet.readthedocs.io/en/latest/ 
   doc_source_url: https://github.com/chardet/chardet/blob/master/docs/index.rst
```

Comment:
as there is already a doc_url, the doc_source_url tag is superflous

Reply:
It seems that the changelog doesn't contain breaking changes. But testing with downstream packages can be helpful.

Reply:
Okay, I can remove this line.